features/admin.connections.v1/components/create/custom-authentication-create-wizard.tsx

```diff
@@ -0,0 +1,1136 @@
+     *
+     * @param identityProvider - identity provider object.
+     */
+    const createExternalAuthenticator = (identityProvider: ConnectionInterface) => {
+        createConnection(identityProvider)
+            .then((response: AxiosResponse<ConnectionInterface>) => {
+                eventPublisher.publish("connections-finish-adding-connection", {
+                    type: _componentId + "-" + kebabCase(selectedAuthenticator)
+                });
+                dispatch(
+                    addAlert({
+                        description: t("authenticationProvider:notifications." + "addIDP.success.description"),
+                        level: AlertLevels.SUCCESS,
+                        message: t("authenticationProvider:notifications." + "addIDP.success.message")
+                    })
+                );
+                // The created resource's id is sent as a location header.
+                // If that's available, navigate to the edit page.
+                if (!isEmpty(response.headers.location)) {
+                    const location: string = response.headers.location;
+                    const createdIdpID: string = location.substring(location.lastIndexOf("/") + 1);
+
+                    onIDPCreate(createdIdpID);
+
+                    return;
+                }
+                onIDPCreate();
+            })
+            .catch((error: AxiosError) => {
+                const identityAppsError: IdentityAppsError = ConnectionUIConstants.ERROR_CREATE_LIMIT_REACHED;
+
+                if (error.response.status === 403 && error?.response?.data?.code === identityAppsError.getErrorCode()) {
+                    setAlert({
+                        code: identityAppsError.getErrorCode(),
+                        description: t(identityAppsError.getErrorDescription()),
+                        level: AlertLevels.ERROR,
+                        message: t(identityAppsError.getErrorMessage()),
+                        traceId: identityAppsError.getErrorTraceId()
+                    });
+                    setTimeout(() => setAlert(undefined), 4000);
+
+                    return;
+                }
+
+                if (error?.response.status === 500 && error.response?.data.code === "IDP-65002") {
+                    setAlert({
+                        description: t("authenticationProvider:notifications." + "addIDP.serverError.description"),
+                        level: AlertLevels.ERROR,
+                        message: t("authenticationProvider:notifications." + "addIDP.serverError.message")
+                    });
+                    setTimeout(() => setAlert(undefined), 8000);
+
+                    return;
+                }
+
+                if (error.response && error.response.data && error.response.data.description) {
+                    setAlert({
+                        description: t("authenticationProvider:notifications." + "addIDP.error.description", {
```

Comment:
Let's avoid unnecessary concatenation. Please check and update the other places as well.

```suggestion
                        description: t("authenticationProvider:notifications.addIDP.error.description", {
```

Reply:
This is updated.